pages/index.vue

```diff
@@ -20,10 +22,12 @@
             :is-horizontal="false"
         />
 
-        <divider-way-finder
-            class="divider"
-            color="visit"
-        />
+        <section-wrapper theme="divider">
+            <divider-way-finder
+                class="divider"
+                color="visit"
+            />
+        </section-wrapper>
 
         <banner-featured
```

Comment:
wrap this too

Reply:
But shouldn't we skip banners when using section-wrapper?

Reply:
only BannerHeader and BannerText skip the section-wrapper